functional/MBCS_Tests/pattern_matching_instanceof/build.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0"?>
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<project name="MBCS_Tests/pattern_matching_instanceof" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>
+	MBCS_Tests/pattern_matching_instanceof
+	</description>
+
+	<import file="${TEST_ROOT}/functional/MBCS_Tests/buildDependencies.xml"/>
+
+	<!-- set global properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/functional/MBCS_Tests/pattern_matching_instanceof" />
+
+	<!--Properties for this particular build-->
+	<property name="src" location="./src" />
+	<property name="build" location="./bin" />
+	
+	<target name="init">
+		<mkdir dir="${DEST}" />
+		<mkdir dir="${build}" />
+	</target>
+
+	<target name="compile" depends="init" description="Using java${JDK_VERSION} to compile the source">
```

Comment:
This is for jdk17+? If that is the case I would suggest do a if condition in build task. That is 

```
	<target name="build">
		<if>
			<not>
				<matches string="${JDK_VERSION}" pattern="^(8|9|10|11|12|13|14|15|16)$$" />
			</not>
			<then>
				<antcall target="clean" inheritall="true" />
			</then>
		</if>
	</target>
```